skll/config/utils.py

```diff
@@ -134,7 +133,7 @@ def _munge_featureset_name(featureset: FeatureSet) -> str:
 
     Parameters
     ----------
-    featureset : skll.data.FeatureSet
+    featureset : Union[skll.data.FeatureSet, str]
```

Comment:
This doesn't match the type above. also if it's not a FeatureSet, maybe change the param's name?

Reply:
Good catch! Will fix.